test/NuGet.Core.Tests/NuGet.Commands.Test/RestoreRunnerTests.cs

```diff
@@ -793,7 +793,7 @@ public async Task RestoreRunner_BasicPackageDownloadRestoreAsync()
                 var packageSource = new DirectoryInfo(Path.Combine(workingDir, "packageSource")); packageSource.Create();
                 var project1Folder = new DirectoryInfo(Path.Combine(workingDir, "projects", project1)); project1Folder.Create();
                 // set up project1
-                var projectSpec = JsonPackageSpecReader.GetPackageSpec(packageSpec, project1, Path.Combine(project1Folder.FullName, "packageSpec.json"));
```

Comment:
I changed some test code around and it caused these tests to fail. 

These simple changes fix them.